applications/klighd-vscode/src/extension.ts

```diff
@@ -63,6 +70,18 @@ export function activate(context: vscode.ExtensionContext): void {
         )
     );
 
+    // Command for the user to remove all data stored by this extension. Allows
+    // the user to reset changed synthesis options etc.
+    context.subscriptions.push(
+        vscode.commands.registerCommand(command.clearData, () => {
+            StorageService.clearAll(mementoForPersistence);
+            // Inform the user that the deletion does not effect previously initialized options on the LS.
+            vscode.window.showInformationMessage(
+                "Stored data has been deleted. Diagram options will be reset to default after a reload."
```

Comment:
This could also notify the core that all options should be reset to their default value right now, making a window reload obsolete.

Reply:
This *tiny* improvement only required changes to 13 files. Hopefully users notice it :D

*insert "this is fine" meme*